unepwcmc/action-agenda
# 17 devise styling

We only have a [design for the log in page](https://xd.adobe.com/view/85f38f01-ca5a-4c6b-a393-28b15a7e278b-d0a0/screen/c651bdf9-a319-488f-ad72-502e860a704a/specs), but I've also styled the registration, forgotten password, unlock and confirmation pages

# Testing

Make sure you're logged out, then:
1. visit [/criteria/new](http://localhost:3000/criteria/new), which should redirect to the sign in page with an error message
2. fill in the form with a bad email and password (i.e. not test@test.com / password or test2@test.com / password) and try to sign in to reveal another error message
3. go to [/users/sign_up](http://localhost:3000/users/sign_up)
4. go to [/users/password/new](http://localhost:3000/users/password/new)
5. go to [/users/confirmation/new](http://localhost:3000/users/confirmation/new)
6. go to [/users/unlock/new](http://localhost:3000/users/unlock/new)
